SocketRocket/SRSecurityPolicy.h

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2012 Square Inc.
```

Comment:
Nit: Since this is new code added and is not part of the original codebase - let's use the copyright statement from `SRSecurityPolicyBuilder`.